src/disk_io_thread.cpp

```diff
@@ -2084,22 +2086,27 @@ namespace libtorrent
 		DLOG("kick_hasher: %d - %d (piece: %d offset: %d)\n"
```

Comment:
There are other places that offset is read without checking `hashing`, in particular `try_flush_hashed`.